Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesTooltipTrait.swift

```diff
@@ -75,9 +75,8 @@ internal class AppcuesTooltipTrait: AppcuesStepDecoratingTrait, AppcuesWrapperCr
         }
     }
 
-    func addBackdrop(backdropView: UIView, to wrapperController: UIViewController) {
-        wrapperController.view.insertSubview(backdropView, at: 0)
-        backdropView.pin(to: wrapperController.view)
+    func getBackdrop(for wrapperController: UIViewController) -> UIView? {
+        return (wrapperController as? ExperienceWrapperViewController<TooltipWrapperView>)?.bodyView.backdropView
```

Comment:
fyi - test coverage report caught this issue - I had initially had this line as 
```
(wrapperController as? ExperienceWrapperViewController)
```
but turns out that returns nil here! It needs to be the specialized version used in L46 above with the TooltipWrapperView 😅